plugin.audio.euskarazko-irratiak/default.py

```diff
@@ -0,0 +1,72 @@
+# https://docs.python.org/2.7/
+import os
+import sys
+import urllib
+import urlparse
+# http://mirrors.kodi.tv/docs/python-docs/
+import xbmcaddon
+import xbmcgui
+import xbmcplugin
+# http://docs.python-requests.org/en/latest/
+import requests
+
+JSON_URL = 'https://raw.githubusercontent.com/aldatsa/plugin.audio.euskarazko-irratiak/master/streams/streams.json'
+
+def build_url(query):
+    base_url = sys.argv[0]
+    return base_url + '?' + urllib.urlencode(query)
+
+def get_streams(url):
+    """
+    Get the list of audio/streams.
+
+    :param url: str
+    :return: list
+    """
+
+    data = requests.get(url)
+    return data.json().get('streams')
+
+def list_streams(streams):
+    stream_list = []
+    # iterate over the contents of the dictionary songs to build the list
+    for stream in streams:
+        # create a list item using the stream's name for the label
+        li = xbmcgui.ListItem(label=stream['name'], thumbnailImage=stream['logo'])
```

Comment:
For listitem art you might want to consider in the future the setArt method:
https://codedocs.xyz/xbmc/xbmc/group__python__xbmcgui__listitem.html#gad3f9b9befa5f3d2f4683f9957264dbbe

Both `iconImage` and  `thumbnailImage` are deprecated